cmd/centrifuge_cmd_test.go

```diff
@@ -0,0 +1,38 @@
+// +build cmd
```

Comment:
Any specific reason why new build tag instead of unit test. I think these belong to unit test IMO